Makefile

```diff
@@ -11,53 +11,55 @@
 # make push-helm
 # make REGISTRY=gcr.io/$PROJECT_ID push-images -j$(nproc)
 # make install-chart
-# 
+#
 # Generate Files
 # make all-protos
-# 
+#
 # Building
 # make all -j$(nproc)
-# 
+#
 # Access monitoring
 # make proxy-prometheus
 # make proxy-grafana
-# 
+#
 # Run those tools
 # make run-backendclient
 # make run-frontendclient
 # make run-clientloadgen
-# 
+#
 # Teardown
 # make delete-mini-cluster
 # make delete-gke-cluster
-# 
+#
 ## http://makefiletutorial.com/
 
 BASE_VERSION = 0.4.0
 VERSION ?= $(BASE_VERSION)-$(shell git rev-parse --short=7 HEAD)
 
-PROTOC_VERSION = 3.6.1
+PROTOC_VERSION = 3.7.0
+GOLANG_VERSION = 1.12
+HELM_VERSION = 2.13.0
+KUBECTL_VERSION = 1.13.0
+
 PROTOC_RELEASE_BASE = https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/protoc-$(PROTOC_VERSION)
 GO = go
 GO_BIN := $(GOPATH)/bin
 GO_BUILD_COMMAND = CGO_ENABLED=0 GOOS=linux $(GO) build -a -installsuffix cgo .
 TOOLCHAIN_DIR = build/toolchain
 TOOLCHAIN_BIN = $(TOOLCHAIN_DIR)/bin
-TOOLCHAIN_GOLANG_DIR = $(TOOLCHAIN_DIR)/golang
 PROTOC := $(TOOLCHAIN_BIN)/protoc
 PROTOC_INCLUDES := $(TOOLCHAIN_DIR)/include/
 GCP_PROJECT_ID = "Set $GCP_PROJECT_ID in your bashrc."
 GCP_PROJECT_FLAG = --project=$(GCP_PROJECT_ID)
 REGISTRY := gcr.io/$(GCP_PROJECT_ID)
 TAG := $(VERSION)
+ALTERNATE_TAG := dev
```

Comment:
Whats the motivation to have both tags (release and dev)?

Reply:
Convenience. Refer to the :dev tag when you want to update new images. the :short_sha images are meant to be immutable.